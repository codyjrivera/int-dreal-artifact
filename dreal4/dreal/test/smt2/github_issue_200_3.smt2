;; From https://github.com/dreal/dreal4/issues/200
(set-logic QF_NRA)
(set-option :worklist-fixpoint true)
(declare-const r31415926 Real)
(declare-const r31415927 Real)
(push 1)
(check-sat)
(pop 1)
(assert (not (< r31415926 r31415927)))
(check-sat)
(declare-const r4 Real)
(push 1)
(assert false)
(check-sat)
(check-sat)
(pop 1)
(assert (or (= (+ r4 23531254.0) 0.0) (not (< r31415926 r31415927))))
(check-sat)
