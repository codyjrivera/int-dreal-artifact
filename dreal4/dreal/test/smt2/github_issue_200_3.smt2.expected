delta-sat with delta = 0.001
delta-sat with delta = 0.001
unsat
unsat
delta-sat with delta = 0.001
